### PR TITLE
Mark 2 existing indirect jump resolving techniques as deprecated

### PR DESCRIPTION
**Pull request by tyb0807**
@ltfish I'm not sure if we still try to resolve the indirect jumps after displaying the warning message or not since these 2 techniques are currently broken?

### CODE REVIEW

File: angr/analyses/cfg/cfg_accurate.py [hunk 1]
```diff
@@ -208,6 +208,10 @@ def __init__(self,
             else:
                 raise AngrCFGError('Unsupported type of the `starts` argument.')
 
+        if enable_advanced_backward_slicing or enable_symbolic_back_traversal:
+            l.warning("`advanced backward slicing` and `symbolic back traversal` are deprecated. Nothing will happen now.")
```
Comment (ltfish) [inline angr/analyses/cfg/cfg_accurate.py]: "Nothing will happen now" is inaccurate. These two options will still work, right?

Comment (tyb0807) [reply angr/analyses/cfg/cfg_accurate.py]: No the old techniques are never called. Lemme put them back

**Comment by tyb0807** [comment]
I guess it is better now

**Comment by ltfish** [comment]
Thanks!

**Pull request merged**